qemu/tests/blockdev_commit_powerdown.py

```diff
@@ -0,0 +1,66 @@
+from virttest import env_process
+
+from provider import job_utils
+from provider import backup_utils
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitHotunplug(BlockDevCommitTest):
+
+    def commit_snapshot_and_destory_vm(self):
+        device = self.params.get("device_tag")
+        device_params = self.params.object_params(device)
+        snapshot_tags = device_params["snapshot_tags"].split()
+        self.device_node = self.get_node_name(device)
+        options = ["base-node", "top-node", "speed"]
+        arguments = self.params.copy_from_keys(options)
+        device = self.get_node_name(snapshot_tags[-1])
+        commit_cmd = backup_utils.block_commit_qmp_cmd
+        cmd, args = commit_cmd(device, **arguments)
+        self.main_vm.monitor.cmd(cmd, args)
+        self.main_vm.destroy(gracefully=False)
```

Comment:
suggest to check job started before vm destroy (job_utils.check_block_jobs_started)

Reply:
suggest to check job cancelled after vm destroy (job_utils.get_event_by_condition, event: BLOCK_JOB_CANCELLED)

Reply:
Ok,added `check_block_jobs_started()` here, but not check if job is cancelled, since I think it will be enough if qemu-kvm quit without any coredump and errors.

Reply:
U are right, qemu quits after vm destroy, nothing more needs to be checked.